Sources/Testing/Events/Recorder/Event.AdvancedConsoleOutputRecorder.swift

```diff
@@ -25,6 +25,15 @@ extension Event {
       }
     }
     
+    /// Context for storing data across events during test execution.
+    private struct Context: Sendable {
```

Comment:
Style: we underscore-prefix `private` declarations
```suggestion
    private struct _Context: Sendable {
```